SalaryService.Application/Dtos/CompensationCeoListDto.cs

```diff
@@ -0,0 +1,23 @@
+﻿public class CompensationCeoListDto
```

Comment:
We have an agreement that we don't use any role information in the code. So please think about how to avoid such names